src/constants/colors.js

```diff
@@ -3,5 +3,6 @@ export const BLACK_OPACITY = BLACK + '50';
 export const WHITE = '#ffffff';
 export const RED = '#ff3738';
 export const GRAY = '#333';
+export const BLUE = 'steelblue'

```

Comment:
Don't forget about semicolons. This can be fixed by installing eslint rules and press autoformatter hotkey